lib/rspecq/queue.rb

```diff
@@ -241,11 +244,36 @@ def flaky_jobs
       requeued - @redis.hkeys(key_failures)
     end
 
+    # Returns the number of failures that will trigger the build to fail-fast.
+    # Returns 0 if this feature is disabled and nil if the Queue is not yet
+    # published
+    def fail_fast
+      return nil unless published?
+
+      @fail_fast ||= Integer(@redis.hget(key_queue_config, 'fail_fast'))
+    end
+
+    # Returns true if the number of failed tests, has surpassed the threshold
+    # to render the run unsuccessful and the build should be terminated.
+    def build_failed_fast?
+      return false if fail_fast.zero?
```

Comment:
Here we do not account for the fact that `fail_fast` might return `nil`. I believe we should do so:

```ruby
if fail_fast.nil? || fail_fast.zero?
  return false
end
```

Reply:
Nice catch

Reply:
Fixed